gammapy/makers/utils.py

```diff
@@ -185,11 +217,58 @@ def _map_spectrum_weight(map, spectrum=None):
     return map * weights.reshape(shape.astype(int))
 
 
+def _evaluate_bkg(pointing, ontime, bkg, geom, use_region_center, obstime, d_omega):
+    """
+    Evaluate the background IRF on a given geometry.
+
+    Parameters
+    ----------
+    pointing :  `~gammapy.data.FixedPointingInfo` or `~astropy.coordinates.SkyCoord`
+        Observation pointing.
+    ontime : `~astropy.units.Quantity`
+        Observation ontime. i.e. not corrected for deadtime
+    bkg : `~gammapy.irf.Background3D`
+        Background rate model.
+    geom : `~gammapy.maps.WcsGeom`
+        Reference geometry.
+    use_region_center : bool, optional
+        For geom as a `~gammapy.maps.RegionGeom`. If True, consider the values at the region center.
+        If False, average over the whole region.
+        Default is True.
+    obstime : `~astropy.time.Time`
+        Observation time to use.
+    d_omega : 'astropy.units.Quantity'
+        Solid angle of the image geometry.
+
+    Returns
+    -------
+    evaluated_bkg : `numpy.ndarray`
+        Background IRF evaluated on the provided geometry
+
+    """
+    coords = _get_fov_coords(
+        pointing=pointing,
+        irf=bkg,
+        geom=geom,
+        use_region_center=use_region_center,
+        #  TODO `obstime` should be provided in future versions of gammapy.
+        #  TODO Using None is deprecated. if else to be removed.
```

Comment:
@registerrier : I am wondering whether one should be more strict on this, in preparation of the v2.0. The warning message in `_get_fov_coords` should mention that it will be a mandatory argument from v2.0. Any preference?